<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmImageSpliceClass</class>
 <widget class="QDialog" name="frmImageSpliceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>655</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1160</width>
    <height>655</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1160</width>
    <height>655</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>图像拼接</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(220, 220, 220);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="1" column="2">
    <layout class="QGridLayout" name="imageLayout">
     <property name="spacing">
      <number>1</number>
     </property>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #bcc2bc;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基本设置</string>
      </attribute>
      <widget class="QGroupBox" name="groupCamera">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>18</y>
         <width>427</width>
         <height>244</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);
color: rgb(85, 0, 255);</string>
       </property>
       <property name="title">
        <string>图像对象</string>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>36</y>
          <width>411</width>
          <height>201</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">border: 1px solid;
border-color: rgba(173, 173, 173, 155);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QTableWidget" name="tableWidgetImage">
              <property name="font">
               <font>
                <family>微软雅黑</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QTableWidget          
{
    background-color: #b7c7af;   
    border:none;
    gridline-color: rgba(20, 25, 155,50);
    color: rgba(20, 20, 20,200);
}
QTableWidget::item::selected
{   
    background-color: rgba(180, 180, 215,255); 
}
QScrollBar:vertical{ 
		background:#FFFFF2;
		padding-top:20px;
		padding-bottom:20px;
		padding-left:2px;
		padding-right:2px;
		border-left:1px solid #d7d7d7;}
QScrollBar::handle:vertical{
		background:#dbdbdb;
		border-radius:6px;
		min-height:20px;}
QScrollBar::handle:vertical:hover{
		background:#d0d0d0;}
QScrollBar::add-line:vertical{
		background:url(:/resource/down.png) center no-repeat;}
QScrollBar::sub-line:vertical{
		background:url(:/resource/up.png) center no-repeat;}
QScrollBar:horizontal{ 
		background:#FFFFF2;
		padding-top:20px;
		padding-bottom:20px;
		padding-left:2px;
		padding-right:2px;
		border-left:1px solid #d7d7d7;}
QScrollBar::handle:horizontal{
		background:#dbdbdb;
		border-radius:6px;
		min-height:20px;}
QScrollBar::handle:horizontal:hover{
		background:#d0d0d0;}
QScrollBar::add-line:horizontal{
		background:url(:/resource/right.png) center no-repeat;}
QScrollBar::sub-line:horizontal{
		background:url(:/resource/left.png) center no-repeat;}</string>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="gridStyle">
               <enum>Qt::DashLine</enum>
              </property>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>29</number>
              </attribute>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>339</number>
              </attribute>
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>30</number>
              </attribute>
              <column>
               <property name="text">
                <string>图像链接地址</string>
               </property>
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>9</pointsize>
                </font>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="styleSheet">
            <string notr="true">border: 1px solid;
border-color: rgba(173, 173, 173, 155);
background-color: #b7c7a7;   </string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>13</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="btnAddImage">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget{ background-color: rgba(9, 42, 47, 200);}
QWidget{ color: rgb(235, 235, 235);}
QWidget{ border: 2px groove gray;border-radius: 5px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgb(9, 62, 52);}
QPushButton:pressed{background-color: rgb(9, 82, 72);}</string>
              </property>
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnDeleteImage">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget{ background-color: rgba(97, 63, 56, 200);}
QWidget{ color: rgb(235, 235, 235);}
QWidget{ border: 2px groove gray;border-radius: 5px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgb(97, 83, 76);}
QPushButton:pressed{background-color: rgb(117, 83, 96);}</string>
              </property>
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupCamera_2">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>276</y>
         <width>427</width>
         <height>73</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #dfd5cf, stop: 0.15 #acbbac, stop: 0.85 #acbbac,stop:1 #dfd5cf);
color: rgb(85, 0, 255);</string>
       </property>
       <property name="title">
        <string>拼接参数</string>
       </property>
       <widget class="QDoubleSpinBox" name="spinRegistrationResol">
        <property name="geometry">
         <rect>
          <x>114</x>
          <y>38</y>
          <width>73</width>
          <height>27</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>值越小拼接速度越快，但匹配点变少</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget{color: rgb(85, 0, 155);background-color: rgb(180, 180, 210);}
QDoubleSpinBox::up-button{
    border:transparent;
}
QDoubleSpinBox::down-button{
    border:transparent;
}
QDoubleSpinBox::up-arrow{
    image:url(:/resource/up.png);
}
QDoubleSpinBox::down-arrow{
    image:url(:/resource/down.png);
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="minimum">
         <double>0.100000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.350000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>42</y>
          <width>101</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(30, 30, 30);
background: transparent;</string>
        </property>
        <property name="text">
         <string>配准分辨率：</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>42</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>83</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
       </property>
       <property name="text">
        <string>单元描述：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDescribe">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>357</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExecute">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>37</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget{ background-color: rgb(85, 135, 127);}
QWidget{ color: rgb(235, 235, 235);}
QWidget{ border: 2px groove gray;border-radius: 8px;padding: 2px 4px;}
QPushButton:hover{ background-color: rgb(85, 155, 127);}
QPushButton:pressed{background-color: rgb(85, 175, 127);}</string>
       </property>
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>6</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>3</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sub_res.qrc"/>
  <include location="../ImageView/sub_res.qrc"/>
 </resources>
 <connections/>
</ui>
